packages/desktop-client/src/components/reports/reports/Spending.tsx

```diff
@@ -139,48 +139,115 @@ export function Spending() {
       <View
         style={{
           flexDirection: 'row',
+          alignItems: 'center',
+          padding: 20,
+          paddingBottom: conditions.length > 0 ? 0 : 10,
           flexShrink: 0,
         }}
       >
         <View
           style={{
-            padding: 20,
-            flexShrink: 0,
+            alignItems: 'center',
+            flexDirection: 'row',
+            marginRight: 5,
           }}
         >
-          {conditions && (
-            <View style={{ flexDirection: 'row' }}>
-              <FilterButton
-                onApply={onApplyFilter}
-                compact={false}
-                hover={false}
-                exclude={['date']}
-              />
-              <Tooltip
-                placement="bottom start"
-                content={<Text>Save compare and filter options</Text>}
-                style={{
-                  ...styles.tooltip,
-                  lineHeight: 1.5,
-                  padding: '6px 10px',
-                  marginLeft: 10,
-                }}
-              >
-                <Button
-                  type="primary"
-                  style={{
-                    marginLeft: 10,
-                  }}
-                  onClick={saveFilter}
-                  disabled={filterSaved ? true : false}
-                >
-                  {filterSaved ? 'Saved' : 'Save'}
-                </Button>
-              </Tooltip>
-              <View style={{ flex: 1 }} />
-            </View>
+          <strong
+            style={{
+              paddingRight: 5,
+            }}
+          >
+            Compare
+          </strong>
+          <Select
+            value={compare}
+            buttonStyle={{
+              fontWeight: 'bold',
+            }}
+            onChange={e => {
+              setCompare(e);
+              if (mode === 'lastMonth') setMode('twoMonthsPrevious');
+              if (mode === 'twoMonthsPrevious') setMode('lastMonth');
+            }}
+            options={[
+              ['thisMonth', 'this month'],
+              ['lastMonth', 'last month'],
+            ]}
+          />
+          <strong
+            style={{
+              paddingRight: 10,
+              paddingLeft: 5,
+            }}
+          >
+            to the:
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏removing "the" makes the entire sentence sound more correct. I.e. "Compare last month to: average" 
```suggestion
            to:
```

Reply:
My reason for using "month previous" and "the" was because it could be confusing.

Example:
"Compare last month to: previous month" - could be interpreted as comparing July 2024 to July 2024 because "last month" and "previous month" mean the same thing
"Compare last month to the: month previous" - gives a bit more clarity that we are comparing July 2024 to June 2024.

Reply:
I see. Thanks for explaining!

Not worth spending much more time on this IMO if we can eventually (not this PR) land on the feature request: https://github.com/actualbudget/actual/pull/3166#discussion_r1702791986